src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/slurm/SlurmJobManager.groovy

```diff
@@ -239,7 +251,7 @@ class SlurmJobManager extends GridEngineBasedJobManager {
             jobInfo.userGroup = jobResult["group"]
             jobInfo.jobGroup = jobResult["group"]
             jobInfo.priority = jobResult["priority"]
-            jobInfo.executionHosts = jobResult["nodes"] as List<String>
+            jobInfo.executionHosts = [jobResult["nodes"] as String]
```

Comment:
jobResult["nodes"] returns a String like "compute-node12". I am assume it would return them separated with "," or something if there was more than one. But i was not able to get produce an example on our cluster with more than one node. I could just guess and add a split on (",") if u think that is a better solution

The Problem reason for the "as String" is that without it u get: Cant cast List<Object> to List<String>

Reply:
That's a strange message. `Can't cast List to List`?

Man, no test! No example output/json to parse to understand what kind of value is expected there. Really hard to tell whether there would really be a List parsed. The error message suggests, it is a List ("Can't cast List ..."). But the message itself -- why does it have a problem to cast a list to a list. Maybe because of the generic type? Furthermore, you cast to String now and it works? 

Better do not guess here. As long as you return a `List<String>` it's fine. If you wish you can look into the SLURM docs to learn more. Maybe that also clarifies the problem with JSON output and the cast error. 

Finally, we don't use any multi-node jobs in Roddy and probably also not in OTP, do we? In that case, just ignore the problem.

Reply:
Actually, can you rename `jobResult` to `jsonEntry`, or so? I think that would be more fitting.

Reply:
i changed the formatting since that one got lost in my first message sorry for that

Reply:
Yeah we dont have multi-node jobs